Bug 1537890 format numbers in tooltips

DraciVik opened the pull request:
Formats number in tooltips for bug 1537890.
One problem I'm running into is that when I run the testing - Yarn lint - it shows me errors that have nothing to do with the files I changed.
To be sure it's not me I switched to master and ran the command there and it shows the same errors.
Thank you for your patience @sarah-clements !

sarah-clements commented:
> One problem I'm running into is that when I run the testing - Yarn lint - it shows me errors that have nothing to do with the files I changed.
> To be sure it's not me I switched to master and ran the command there and it shows the same errors.

I'm not seeing a problem since all of the tests passed. However, in the future you should use `rebase` instead of pull/merge. Specifically, you want to sync with upstream: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

When you rebase, there isn't additional commits like you see with merge/pull so it keeps the branch very clean with only relevant commits. All commits will get squashed when I merge it, but I suggest you read up on this command for your own benefit. :)

sarah-clements commented:
You can work on the eslint task that's outlined on the the project Outreachy page, if you haven't done so already, and then start to finalize your application.

DraciVik commented:
> 
> 
> > One problem I'm running into is that when I run the testing - Yarn lint - it shows me errors that have nothing to do with the files I changed.
> > To be sure it's not me I switched to master and ran the command there and it shows the same errors.
> 
> I'm not seeing a problem since all of the tests passed. However, in the future you should use `rebase` instead of pull/merge. Specifically, you want to sync with upstream: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork
> 
> When you rebase, there isn't additional commits like you see with merge/pull so it keeps the branch very clean with only relevant commits. All commits will get squashed when I merge it, but I suggest you read up on this command for your own benefit. :)

Thank you! I will do it with rebase from now on. I had a hunch that there has to be a better way :). 

> 
> 
> You can work on the eslint task that's outlined on the the project Outreachy page, if you haven't done so already, and then start to finalize your application.

Yes, I'm almost done. You will have it by Monday for review so you can correct me for my inevitable mistakes. :) 

Once again - Thank you for your patience.
